dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -66,15 +67,20 @@ def report_abuse
         return
       end
 
-      send_abuse_report(params[:name], params[:email], params[:age], params[:abuse_url])
+      send_abuse_report(
+        current_user&.name || '',
```

Comment:
When we submit the abuse report, we handle if the name field is an empty string, so I've made sure we include an empty string here instead of nil:

https://github.com/code-dot-org/code-dot-org/blob/088dea4cb75208d851de40b88be8bb23f2deb2e3/dashboard/app/controllers/report_abuse_controller.rb#L184

Reply:
Thanks! I was curious about this.